Intersect.Client/MonoGame/File Management/MonoContentManager.cs

```diff
@@ -244,6 +245,23 @@ public override void LoadSounds()
                     )
                 );
             }
+
+            // If we have a sound index file, load from it!
+            if (File.Exists(Path.Combine("resources", "packs", "sound.index")))
+            {
+                SoundPacks = new AssetPacker(Path.Combine("resources", "packs", "sound.index"), Path.Combine("resources", "packs"));
+                foreach(var item in SoundPacks.FileList)
+                {
+                    if (!mSoundDict.ContainsKey(RemoveExtension(item)))
```

Comment:
Should we be adding to this dictionary as lower case like the music dictionary below?

Reply:
Would be more consistent, but if Panda is going to change everything around that might not be the case anymore? I think he wanted to make stuff case insensitive?